tiledb/sm/subarray/subarray.cc

```diff
@@ -420,6 +527,73 @@ Status Subarray::add_range_var_by_name(
   return add_range_var(dim_idx, start, start_size, end, end_size);
 }
 
+void Subarray::get_label_range(
+    const std::string& label_name,
+    uint64_t range_idx,
+    const void** start,
+    const void** end,
+    const void** stride) const {
+  auto dim_idx = array_->array_schema_latest()
+                     .dimension_label_reference(label_name)
+                     .dimension_id();
+  if (!label_range_subset_[dim_idx].has_value() ||
```

Comment:
NIT: This could be a separate function.

Reply:
I waffled back and forth on this one when I was writing the code, but I ultimately don't think the extra indirection is worth it.